esmvaltool/recipes/recipe_climwip_test_performance_sigma.yml

```diff
@@ -96,15 +75,24 @@ preprocessors:
       mask_out: sea
     extract_region:
       start_longitude: 0.0
-      end_longitude: 39.0
+      end_longitude: 60.0
       start_latitude: 30.0
-      end_latitude: 76.25
+      end_latitude: 80.0
```

Comment:
I'm fine with changing the bounds, but this does not seem to be part of the bugfix. Perhaps you can extend the description of the PR?

Reply:
If there is less models the region needs to be larger for the perfect model test to reliably work - since I removed models to speed things up I had to make the region larger. I will add a description